src/routes/solid-router/installation-and-setup.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Installation and Setup
+---
+
+A router lets you change your view based on the URL in the browser. This allows your "single-page" application to simulate a traditional multipage site. To use Solid Router, you specify components called Routes that depend on the value of the URL (the "path"), and the router handles the mechanism of swapping them in and out.
+
+Solid Router is a universal router for SolidJS - it works whether you're rendering on the client or on the server. It was inspired by and combines paradigms of React Router and the Ember Router. Routes can be defined directly in your app's template using JSX, but you can also pass your route configuration directly as an object. It also supports nested routing, so navigation can change a part of a component, rather than completely replacing it.
+
```

Comment:
I'm not sure i love this but if we have to have it, adding the links to the relevant pages would be a good idea